docs/source/ops/connect/helm.rst

```diff
@@ -3,17 +3,17 @@
 
 .. _connect-helm-chart:
 
-Connect Helm Chart
+Helm Chart
 ==================
```

Comment:
```suggestion
Daml Helm Chart
===============
```
The point is always avoiding linter errors but I also added `Daml` back to be consistent with the edit you made before.